README.md

```diff
@@ -218,6 +218,25 @@ Example:
 * !cmd add df "df -h"
 * !cmd add whoami "echo You are $MATRIX_USER in room $MATRIX_ROOM."
 
+### Astronomy Picture of the Day
+
+Upload and send latest astronomy picture of the day to the room.
+See https://apod.nasa.gov/apod/astropix.html
+
+Command:
+
+* !apod - Sends latest Astronomy Picture of the Day to the room
+* !apod YYYY-MM-DD - date of the APOD image to retrieve (ex. !apod 2020-03-15)
+* !apod stats - show information about uri cache
+* !apod clear - clear uri cache (Must be done as admin)
+* !apod help - show command help
+
+API Key:
+
+The module uses a demo API Key which can be replaced by your own api key by setting the environment variable `APOD_API_KEY`. 
```

Comment:
I would prefer this to be a command that bot owner can do. For example:

!apod apikey [apikey]

This way no changes to env variables and dockerfile would be needed.